### DISCUSSION TITLE
Index.html ends up inside a folder after deploying gh-pages branch

### DISCUSSION THREAD

**Discussion by sircaffe** [community/community#45267] [Actions] [answered]
### Select Topic Area

Question

### Body

I've managed to write a pipeline for deploying my project, however, after successfully building it into gh-pages and then deploying it from the gh-pages branch, nothing appears on the page at the address: https://www.sorkofi.github.io/github-profile-viewer/

Analysing the console and element inspection, I figured my `index.html` cannot find the assets files (CSS and JS) becase the `index.html` file is inside a folder with the repository name (github-profile-viewer).

Inside the gh-pages branch nothing seems different about the file tree, it seems like this change happens when github deploys it from there.

My actions workflow: https://github.com/sorkofi/github-profile-viewer/edit/master/.github/workflows/main.yml

`GH-PAGES Branch`
![Screen Shot 2023-01-24 at 16 42 20](https://user-images.githubusercontent.com/115833152/214406775-571cdd75-11b4-4e2c-92f6-25f8496ef8c0.png)

`File tree and HTML file`
![Screen Shot 2023-01-24 at 16 42 03](https://user-images.githubusercontent.com/115833152/214406780-d9f9fbdf-2ee2-4bb7-b5f9-f15b675a218a.png)

`Console Errors` 
![Screen Shot 2023-01-24 at 16 42 49](https://user-images.githubusercontent.com/115833152/214406763-91644e95-6c84-4dc4-9a7a-5ba92a5b23af.png)

**Comment by sircaffe** [comment]
I have absolutely no idea how i'd go for solving this, and if what i pointed out is the actual cause of the issue. Any ideas ?

**Comment by airtower-luna** [comment, marked as answer]
The problem is that your `index.html` references the other files with an *absolute* path, starting with a slash (`/assets/...`). This tells the browser to request exactly that path from the hostname. You need to use a relative path instead, so the browser builds the path relative to the current file. E.g. if `index.html` and `assets/` are in the same directory, you need to use `./assets/...`.

**Reply by sircaffe** [reply]
That did the trick, thank you very much. Such a tiny mistake, i didn't notice it at all
